### DISCUSSION TITLE
[Urgent issue] Side menu disappears and can't reach to the page

### DISCUSSION THREAD

**Discussion by yuxizama** [GitbookIO/community#521] [0. General bugs and questions]
Hi Gitbook team,

My team has encountered 2 major issues with Gitbook since this morning.
First, when we land on the documentation page of any product, for example, https://docs.zama.ai/concrete-ml, the side menu does not display, we lose the navigation and this is a huge issue for users. But then after a refreshment of the page, the menu displays correctly.
![image](https://github.com/GitbookIO/community/assets/157474013/0231019c-7e2c-49d1-9a8a-c2002a03598b)
<img width="1714" alt="Screenshot 2024-03-14 at 15 08 38" src="https://github.com/GitbookIO/community/assets/157474013/b0c458c0-f086-4bdb-8d3e-67c4d2490ae3">

Second, to reach any page of our documentation website, we had this Error 1015 from time to time. We've been doing many changes on our documentation, could it be related?
![image](https://github.com/GitbookIO/community/assets/157474013/c9402bff-0bba-4a3f-aae6-ad96081755b2)

Please help to solve these problems. 
Thank you very much!

**Comment by aikoii** [comment]
Hi @yuxizama! Apologies for the late response—we've had a hectic few weeks. For immediate assistance in the future, feel free to contact support directly. We've resolved the Error 1015 issue, and it seems the side menu is now appearing correctly 🙂. I'll close this, do reach out if I'm missing anything!
